src/server.ts

```diff
@@ -98,11 +98,13 @@ export function createHandler<Context, Methods extends RPCMethods>(
       }
       return createErrorResponse(id, ERROR_CODE.INVALID_REQUEST)
     }
+    const handler = methods[msg.method]
     if (id == null) {
-      onNotification(ctx, msg)
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore params can be undefined
+      handler == null ? onNotification(ctx, msg) : void handler(ctx, msg.params)
```

Comment:
I'm a bit concerned about using `void` here as it would swallow errors thrown in the handler rather than trigger the `onHandlerError()` callback, might be safer to apply a similar logic to the other handlers.